src/shared/components/palette/utils/date.pipe.spec.ts

```diff
@@ -157,18 +159,7 @@ describe('DatePipe', () => {
   });
   // End of tests for EUI-2667.
 
-  /**
-   * GMT to BST (from 00:59:59 GMT going forward to 02:00:00 BST) on the last Sunday in March
-   */
-  it ('should handle GMT to BST transition', () => {
-    let endOfWinter = new Date(2020, 2, 29, 0, 59, 59)
-    let message = datePipe.transform (endOfWinter.toISOString(), 'GMT', 'dd MMMM yyyy HH:mm:ss.SSS');
-    expect(message).toBe('29 March 2020 00:59:59.000');
-    // tick on 1 second
-    endOfWinter.setTime(endOfWinter.getTime() + 1000);
-    message = datePipe.transform (endOfWinter.toISOString(), '+0100', 'dd MMMM yyyy HH:mm:ss.SSS');
-    expect(message).toBe('29 March 2020 02:00:00.000');
-  })
+  // test removed regarding time zone change at end of winter (change from 00:59 -> 02:00 removed as no longer relevant)
```

Comment:
_Question:_
Again, why is this no longer relevant? We _do_ still have clocks going forwards and backwards... does the new component make it redundant? I'm unclear as to why it's not relevant. Can you explain further in this comment, please?

Reply:
Will explain here as well. Because of the use of moment library and therefore the fixed utc timezone of the datetime picker (that was enabled on instruction from Sreekanth), the read date component also needed to have a fixed utc timezone in order for the user to not get reasonably confused during the movement within pages. This means that any time given to either component will be used as the time and there will no longer be unnecessary movements between the time zones (i.e. giving one of the components a time that changes from what the user wants). This also fits in better with all use cases of the datepipe as there were occasions when timezones were changing twice (two hours behind or in front) when moving through the various steps.

Reply:
As discussed, a comment to this effect would be awesome to prevent some future explorer stumbling into a moment-based timezone trap. 😁

Reply:
Done